pick.c

```diff
@@ -173,10 +184,22 @@ main(int argc, char *argv[])
 	choice = selected_choice();
 	tty_restore(1);
 	if (choice != NULL) {
-		printf("%s\n", choice->string);
-		if (output_description)
-			printf("%s\n", choice->description);
-	} else {
+		if ( opt_mark ) { /* if multiple choices, print out all selected strings */
```

Comment:
If multiple choices are always enabled, you could drop the conditional and always iterate over all choices, printing the marked ones. In `case ENTER` just set `mark = 1` there as well then this logic should be applicable to all methods of chosing lines.

Reply:
Yes, it cloud be and opt_mark = 0 case it could return immediately so no way to mess up with more marked lines.